README.md

```diff
@@ -48,3 +48,42 @@ Because Markdown files rendered by GitHub Pages should behave similar to Markdow
   gems:
     - jekyll-relative-links
   ```
+
+## Configuration
+
+You can configure this plugin in `_config.yml` under the `relative_links` key. This is optional and defaults to:
+
+```yml
+relative_links:
+  enabled:     true
```

Comment:
Technically disabling/enabling the plugin is out of scope of the issue being solved, or is this needed for this functionality?

Reply:
(But as this is super useful, I of course would have no problem this making its way into the code)
(Review mechanism might make it look otherwise, was not my intention - should have just commented the code)

Reply:
yes thats correct. it's a separate commit though